drivers/gpio/gpio_mcux_lpc.c

```diff
@@ -230,11 +230,19 @@ static uint32_t attach_pin_to_isr(uint32_t port, uint32_t pin, uint32_t isr_no)
 	/* Code asumes PIN_INT values are grouped [0..3] and [4..7].
 	 * This scenario is true in LPC54xxx/LPC55xxx.
 	 */
+#ifdef CONFIG_BOARD_LPCXPRESSO51U68
```

Comment:
We need to find a better way to handle this, we shouldn't introduce board specific ifdef's in this file.

Reply:
Also, does this make sense on LPC51U68 in that it seems like only has PINT0/1/2/3 so the else case doesn't make any sense.